sdks/typescript/src/apache_beam/utils/service.ts

```diff
@@ -260,7 +260,7 @@ export class JavaJarService extends SubprocessService {
       const tmp = dest + ".tmp" + Math.random();
       return new Promise((resolve, reject) => {
         const fout = fs.createWriteStream(tmp);
-        console.warn("Downloading", urlOrPath);
+        console.info("Downloading", urlOrPath);
```

Comment:
This can take a while (and look like it's hanging) but should only happen the first time, are info logs shown by default?

Reply:
i am seeing info in dataflow, should i revert this change?

Reply:
Let's leave it for now. I'd rather be too verbose (once) than leave people hanging.

Reply:
Sorry, I meant let's leave it as a warning. It should only show up one time per jar.